### PR TITLE
Fixes #690 update population for sensitivity tests

### LINKED ISSUE #690: Investigate build issue on Linux (all failing at the moment)

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

**Comment by pchelle** [comment]
Linux / build (3.6.3, bionic) gets same error as issue #697

File: tests/data/input-data/Pop500_p1p2p3.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Project: Undefined
 #PK-Sim version: 9.0.0
-"IndividualId","Gender","Population","Population Name","Organism|p1 [�mol]","Organism|p2 [�mol]","Organism|p3 [1/min]"
+"IndividualId","Gender","Population","Population Name","Organism|p1 [�mol]","Organism|p2 [�mol]","Organism|p3 [1/min]"
```
[the remark this answers is not in the document]
Comment (pchelle) [reply tests/data/input-data/Pop500_p1p2p3.csv:3]: Unit should be good, does github guess a wrong encoding ?

[the remark this answers is not in the document]
Comment (pchelle) [reply tests/data/input-data/Pop500_p1p2p3.csv:3]: I have read that encoding could be set in the .gitattribute file using `git config gui.encoding utf-8`.

[the remark this answers is not in the document]
Comment (pchelle) [reply tests/data/input-data/Pop500_p1p2p3.csv:3]: Actually, the file is not saved with the proper encoding (`write.csv` default is ISO 8859-1 while we should have enforced UTF-8 in both saving and reading)
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/e23fe79eac281a51a81e70112f5cf42de34d8124/R/population-sensitivity-analysis-task.R#L48

[the remark this answers is not in the document]
Comment (Yuri05) [reply tests/data/input-data/Pop500_p1p2p3.csv:3]: yep.
I think the same happened to the **Pop500_p1p2p3.csv**: it was saved not using UTF-8 .
If you fix the unit and save it as UTF-8, should be fine

**Comment by pchelle** [comment]
@Yuri05 and @msevestre 
I noticed that the file was actually the exported population, thus same applies for `ospsuite::exportPopulationToCSV`
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/f71268db513d0d95c30471bb07b48b5ddd0ff9ac/R/utilities-population.R#L89
Would `loadPopulation` be affected somehow by a change of encoding ?

**Comment by Yuri05** [comment]
finally :) well done!
image

**Pull request merged**